lib/Renderer/View.py

```diff
@@ -65,45 +68,71 @@ def __init__(self, mymap, simulation=None, path=None, windowSize=(1024, 768)):
         
         #play pause step
         self.root.btn_start   = tk.Button(self.frame_btn, text="Play")
-        self.root.btn_step  = tk.Button(self.frame_btn, text="Forward")
-        self.root.btn_step["state"] = tk.DISABLED #disabled until we click on INIT
+        self.root.btn_step    = tk.Button(self.frame_btn, text="Forward")
         
         #number of steps
         self.root.label_step = tk.Label(self.frame_btn, text="Step: 0")
-        self.root.step_scale = tk.Scale(self.frame_btn, label="Step Size (In Seconds)", from_=1, to=100, orient=tk.HORIZONTAL)
+        self.root.step_scale = tk.Scale(self.frame_btn, label="Step Size (In seconds)", from_=1, to=10000, orient=tk.HORIZONTAL)
```

Comment:
I opened a PR about this but maybe you could change it here. I was thinking it would be better if this slider was in minutes instead of seconds

Reply:
Done